app/src/main/java/com/duckduckgo/app/bookmarks/ui/BookmarksActivity.kt

```diff
@@ -76,6 +83,24 @@ class BookmarksActivity : DuckDuckGoActivity() {
         observeViewModel()
     }
 
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+
+        if (requestCode == IMPORT_BOOKMARKS_REQUEST_CODE && resultCode == RESULT_OK) {
+            val selectedFile = data?.data
+            if (selectedFile != null) {
+                viewModel.importBookmarks(selectedFile)
+            }
+        }
+
+        if (requestCode == EXPORT_BOOKMARKS_REQUEST_CODE && resultCode == RESULT_OK) {
```

Comment:
nit: I think it is better to use an `else` statement here. Else we force the reader of the code to understand if both `if`s can be executed.